lib/tree-view-package.js

```diff
@@ -52,6 +52,13 @@ class TreeViewPackage {
     )
   }
 
+  provideTreeView () {
+    return {
+      getSelectedPaths: () => this.getTreeViewInstance().selectedPaths(),
+      getEntry: (path) => this.getTreeViewInstance().entryForPath(path)
```

Comment:
I am one of those people who dislike the `get` and `set` prefixes, since both words are extremely difficult to summarize/describe — look up both words in any English dictionary, and you will see what I mean (see e.g. [get](http://dictionary.cambridge.org/dictionary/english/get), and [set](http://dictionary.cambridge.org/dictionary/english/set)).

**TL; DR** What about changing these `selectedPaths` and `entryForPath`, just as they are named in the underlying API.

Reply:
I agree using the same name as the treeView instance might be helpful